packages/sdk-react/README.md

```diff
@@ -106,11 +127,19 @@ If you want to manually set the initial theme, please pass the config object to
 Initialize TeamsFx and Teams JS SDK, in a development environment, verbose logging message will be printed to console.
 It returns the TeamsFx instance as data.
 If you want to customize TeamsFx like customizing setting, please pass the config object to `useTeamsFx()`.
+> Note: `useTeamsFx()` has been deprecated and will be removed in the future release. Please use `useTeamsUserCredential()` instead as below:
+
+```ts
+const authConfig: TeamsUserCredentialAuthConfig = {
+  clientId: process.env.REACT_APP_CLIENT_ID,
+  initiateLoginEndpoint: process.env.REACT_APP_START_LOGIN_PAGE_URL,
+}
+const { loading, theme, themeString, teamsfx } = useTeamsFx(authConfig);
```

Comment:
Are you going to show the usage of `useTeamsuserCredential()` here?
```suggestion
const { loading, theme, themeString, teamsfx } = useTeamsUserCredential(authConfig);
```

Reply:
Oh, thanks, my mistake

Reply:
Updated